<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>602</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TWICE 갤러리 짤 다운로더 v1.4</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/icon.png</normaloff>res/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>820</width>
      <height>580</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>DC 갤러리</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>갤러리 id :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>44</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>검색어 :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="selectGallery">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>15</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="maxVisibleItems">
       <number>12</number>
      </property>
      <item>
       <property name="text">
        <string>TWICE 갤러리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TWICE TV 갤러리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>나연 갤러리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>정연 갤러리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>모모 갤러리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>사나 갤러리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>지효 갤러리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>미나 갤러리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>다현 갤러리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>채영 갤러리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>쯔위 갤러리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>스트리밍 갤러리</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="editKeyword">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>42</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>213</x>
        <y>18</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>페이지 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editPage">
      <property name="geometry">
       <rect>
        <x>266</x>
        <y>15</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>72</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>제외 단어 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editExcept">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>69</y>
        <width>195</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="onlyBy">
      <property name="geometry">
       <rect>
        <x>217</x>
        <y>45</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>.by만 받기</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="onlyRcmd">
      <property name="geometry">
       <rect>
        <x>298</x>
        <y>45</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>개념글</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>130</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>폴더 경로 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editPath">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>148</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="excptPreview">
      <property name="geometry">
       <rect>
        <x>273</x>
        <y>70</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>프리뷰 제외</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openPath">
      <property name="geometry">
       <rect>
        <x>302</x>
        <y>148</y>
        <width>53</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="folderSeparate">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>125</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>사진별 폴더에 저장</string>
      </property>
     </widget>
     <widget class="QPushButton" name="downloadImg">
      <property name="geometry">
       <rect>
        <x>66</x>
        <y>182</y>
        <width>88</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>다운로드</string>
      </property>
     </widget>
     <widget class="QPushButton" name="downloadCancel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>182</y>
        <width>88</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>91</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>※ 제외단어는 콤마(,)로 구분해 여러단어 입력 가능.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>103</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>페이지 입력 예) 1, 2-10 또는 공백일시 전체페이지</string>
      </property>
     </widget>
     <widget class="QPushButton" name="expandTreeview">
      <property name="geometry">
       <rect>
        <x>362</x>
        <y>90</y>
        <width>31</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>▶</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeView">
      <property name="geometry">
       <rect>
        <x>398</x>
        <y>10</y>
        <width>411</width>
        <height>511</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="retryDownload">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>528</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>재다운로드</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tr_selectDel">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>528</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>선택 삭제</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetTreeview">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>528</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>목록 초기화</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>네이버 포스트/티스토리</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>7</y>
        <width>95</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>다운로드 할 주소: </string>
      </property>
     </widget>
     <widget class="QTextEdit" name="nati_url">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>25</y>
        <width>381</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="downloadCancel_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>480</y>
        <width>88</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
     <widget class="QPushButton" name="downloadImg_2">
      <property name="geometry">
       <rect>
        <x>66</x>
        <y>480</y>
        <width>88</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>다운로드</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>402</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>폴더 경로 :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openPath_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>420</y>
        <width>53</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="folderSeparate_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>402</y>
        <width>125</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>주소별 폴더에 저장</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editPath_2">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>420</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeView_2">
      <property name="geometry">
       <rect>
        <x>398</x>
        <y>10</y>
        <width>411</width>
        <height>511</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="resetTreeview_2">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>528</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>목록 초기화</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tr_selectDel_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>528</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>선택 삭제</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>selectGallery</tabstop>
  <tabstop>editPage</tabstop>
  <tabstop>editKeyword</tabstop>
  <tabstop>onlyBy</tabstop>
  <tabstop>onlyRcmd</tabstop>
  <tabstop>editExcept</tabstop>
  <tabstop>excptPreview</tabstop>
  <tabstop>folderSeparate</tabstop>
  <tabstop>editPath</tabstop>
  <tabstop>openPath</tabstop>
  <tabstop>downloadImg</tabstop>
  <tabstop>downloadCancel</tabstop>
  <tabstop>expandTreeview</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
